transformer_engine/pytorch/transformer.py

```diff
@@ -353,19 +353,15 @@ def __init__(
 
         norm_factor = math.sqrt(self.hidden_size_per_attention_head)
 
-        self.use_flash_attention = (
-            int(os.getenv("NVTE_FLASH_ATTN", "1"))
```

Comment:
Yes that's correct. @tcherckez-nvidia please re-add these lines since we still need this as a way or enabling/disabling FA and only remove it when running the onnx tests